GameData/RealismOverhaul/Engine_Configs/Electric_Engine_Configs/HiPEP_Config.cfg

```diff
@@ -0,0 +1,96 @@
+//	==================================================
+//	HiPEP Gridded Ion Thruster
+//
+//	Manufacturer: NASA Glen Research Center
+//
+//	=================================================================================
+//	HiPEP
+//	JIMO
+//
+//	Dry Mass: 120 Kg
+//	Thrust (SL): ??? N
+//	Thrust (Vac): 0.67 N
+//	Power: 39.3 kW
+//	ISP: ??? SL / 9620 Vac
+//	Burn Time: 30,000 Hours
+//	Chamber Pressure: N/A MPa
+//	Propellant: Xenon
+//	Prop Ratio: N/A
+//	Throttle: 670 mN to 240 mN
+//	Nozzle Ratio: N/A
+//	Ignitions: Infinite
+//	=================================================================================
+
+//	Sources:
+
+//	http://www.grc.nasa.gov/WWW/ion/present/hipep.htm
+//	http://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/20040139476.pdf
+
+//	Used by:
+
+
+//	Notes:
+// 41x91cm
+// kW	mN	Isp
+// 9.7	240	5970
+// 15.9	340	7020
+// 20.2	410	7500
+// 24.4	460	8270
+// 29.6	540	8900
+// 34.6	600	9150
+// 39.3	670	9620
+// Scaling to max values
+//	==================================================
+
+@PART[*]:HAS[#engineType[HIPEP]]:FOR[RealismOverhaulEngines]
+{
+	%title = HiPEP Gridded Ion Thruster
```

Comment:
We really need to be careful with our capitalization here, alternately using HiPEP and HIPEP